www/syncStatus.ts

```diff
@@ -0,0 +1,32 @@
+/// <reference path="../typings/codePush.d.ts" />
+
+"use strict";
+
+/**
+ * Defines the possible result statuses of the window.codePush.sync operation.
+ */
+enum SyncStatus {
+    /**
+     * The application is up to date.
+     */
+    UP_TO_DATE,
+    
+    /**
+     * An update is available, it has been downloaded, unzipped and copied to the deployment folder.
+     * After the completion of the callback invoked with SyncStatus.APPLY_SUCCESS, the application will be reloaded with the updated code and resources.
+     */
+    APPLY_SUCCESS,
+    
+    /**
+     * An optional update is available, but the user declined to install it. The update was not downloaded.
+     */
+    USER_DECLINED,
```

Comment:
I called this state "UPDATE_IGNORED" on the React Native side, but I'm open to either one